docs/dvl/faq.md

```diff
@@ -0,0 +1,52 @@
+**Q1: Do I need to calibrate the IMU each time I power on the DVL?**
+
+**A1:** No. If you use the latest software revision it will be saved in memory.
+
+**Q2: Will the temperature effect the IMU?**
+
+**A2:** Yes, the IMU will perform different under different temperatures then what it was calibrated at.
+
+**Q3: Why does the yaw angle change rapidly?**
+
+**A3:** Yaw angle in the DVL GUI seems to increase or otherwise change rapidly
+This is likely due to the DVL's gyrometer not having being calibrated. See the following for more on the calibration.
+
+[Calibrate the IMU](dead-reckoning.md#starting-dead-reckoning)
+
+If only the velocity output of the DVL is required, this can be ignored, as the yaw angle is not used in the velocity algorithms. It is only if the DVL's dead reckoning functionality will be used that the yaw angle is relevant and calibration is needed.
+
+**Q4: Can we test/operate the DVL in air?**
+
+**A4:** It is possible to test and operate the DVL while out of the water for a short period of time, but the DVL will warm up relatively quickly, and if possible we recommend to avoid this.
```

Comment:
Maybe add something like: "You will not be able to get valid velocity measurements in air", or "the DVL needs to be in water to get valid velocity measurements"

Reply:
[1b12542](https://github.com/waterlinked/waterlinked.github.io/pull/30/commits/1b12542b509c6ac3431599e1125da779633e4dd9)